plasmicpkgs/antd5/src/registerSkeleton.tsx

```diff
@@ -0,0 +1,216 @@
+import { Skeleton, SkeletonProps } from "antd";
+import React from "react";
+import { Registerable, registerComponentHelper } from "./utils";
+
+export type AntdSkeletonProps = SkeletonProps & {
+  type?: 'Basic' | 'Button' | 'Avatar' | 'Input' | 'Image' | 'Node'
+  width?: string | number | (string | number)[]
+  widthTitle?: string
+  rows?: number
+  shape?: 'circle' | 'square'
+  size?: 'large' | 'small' | 'default'
+  customClassName?: string
+}
+
+export function AntdSkeleton(props: AntdSkeletonProps) {
+  const {
+    type,
+    paragraph,
+    avatar,
+    title,
+    shape,
+    size,
+    loading,
+    width,
+    widthTitle,
+    rows,
+    className: rootClassName,
+    customClassName,
+    children,
+  } = props
+
+  if (type === 'Button') {
+    return loading ? (
+      <Skeleton.Button
+        {...props}
+        rootClassName={rootClassName}
+        className={customClassName}
+      />
+    ) : (
+      children
+    )
+  }
+
+  if (type === 'Avatar') {
+    return loading ? (
+      <Skeleton.Avatar
+        {...props}
+        rootClassName={rootClassName}
+        className={customClassName}
+      />
+    ) : (
+      children
+    )
+  }
+
+  if (type === 'Input') {
+    return loading ? (
+      <Skeleton.Input
+        {...props}
+        rootClassName={rootClassName}
+        className={customClassName}
+      />
+    ) : (
+      children
+    )
+  }
+
+  if (type === 'Image') {
+    return loading ? (
+      <Skeleton.Image
+        {...props}
+        rootClassName={rootClassName}
+        className={customClassName}
+      />
+    ) : (
+      children
+    )
+  }
+
+  if (type === 'Node') {
+    return (
+      <Skeleton.Node
+        {...props}
+        rootClassName={props.className}
+        className={customClassName}
+      />
+    )
+  }
+
+  return (
+    <Skeleton
+      {...props}
+      paragraph={paragraph ? { rows, width } : false}
+      avatar={avatar ? { shape, size } : false}
+      title={title ? { width: widthTitle, className: customClassName } : false}
+      rootClassName={rootClassName}
+      className={customClassName}
+    />
+  )
+}
+
+export const skeletonComponentName = "plasmic-antd5-skeleton";
+
+export function registerSkeleton(loader?: Registerable) {
+  registerComponentHelper(loader, AntdSkeleton, {
+    name: skeletonComponentName,
+    displayName: 'Skeleton',
+    props: {
+      children: 'slot',
+      type: {
+        type: 'choice',
+        defaultValue: 'Basic',
+        options: ['Basic', 'Button', 'Avatar', 'Input', 'Image', 'Node'],
+      },
+      active: {
+        type: 'boolean',
+        description: 'Show animation effect',
+        defaultValue: false,
+      },
+      avatar: {
+        type: 'boolean',
+        description: 'Show avatar placeholder',
+        hidden: (ps) => ps.type !== 'Basic',
+        defaultValue: false,
+      },
+      loading: {
```

Comment:
Should be `true` by default, so that when you drag the Skeleton to the canvas, it shows something.